acoular/grids.py

```diff
@@ -611,22 +655,39 @@ def _get_digest(self):
         return digest(self)
 
     def index(self, x, y, z):
-        """Queries the indices for a grid point near a certain co-ordinate.
+        """
+        Return the indices for a grid point near a certain coordinate.
 
-        This can be used to query results or co-ordinates at/near a certain
-        co-ordinate.
+        This can be used to query results or coordinates at or near a certain coordinate. Raises an
+        exception if the given coordinate is outside the grid.
 
         Parameters
         ----------
         x, y, z : float
-            The co-ordinates for which the indices is queried.
+            The coordinates for which the indices is queried.
 
         Returns
         -------
         3-tuple of integers
-            The indices that give the grid point nearest to the given x, y, z
-            co-ordinates from an array with the same shape as the grid.
-
+            The indices that give the grid point nearest to the given x, y, z coordinates from an
+            array with the same shape as the grid.
+
+        Examples
+        --------
+        Check which of the points in a simple 8-point rectangular grid is closest to the point
+        ``(0.5, 0.5, 1.0)``.
+
+        >>> import acoular as ac
+        >>>
+        >>> grid = ac.grids.RectGrid3D()
```

Comment:
All classes in acoular can be directly imported (see the init.py file, which is not empty). Thus, it is always enough to write

```suggestion
        >>> grid = ac.RectGrid3D()
```